### DISCUSSION TITLE
Question about PerTimeInROI units?

### DISCUSSION THREAD

**Discussion by norahmilam** [DeNardoLab/BehaviorDEPOT#21] [Q&A] [answered]
Hi, i just had a quick question about the PerTimeinROI output from Spatial Behavior, is this just total amount of seconds spent in the ROI, or a fraction relative to the total length of the video? Thanks!

**Comment by cjgabrie** [comment, marked as answer]
Hello! Thanks for trying out BehaviorDEPOT!

The PerTimeinROI output is a simple estimation of the fraction of the analyzed video (by frame count) that the animal was in the ROI during the entire session. It is calculated by simply taking the sum of the inROIvector divided by the total number of frames in the session.
